run_dir/design/sequencing_queues.html

```diff
@@ -61,7 +60,7 @@ <h1><span id="page_title">Sequencing queues</span>
 </div>
 
 <script src="/static/js/clipboard.min.js"></script>
```

Comment:
Why does it still need `clipboard.min.js` if its replaced by the new extension?

Reply:
Missed it!